frontend/packages/dev-console/src/components/import/ImportSamplePage.tsx

```diff
@@ -112,10 +112,16 @@ const ImportSamplePage: React.FC<ImportSamplePageProps> = ({ match }) => {
     resources: Resources.Kubernetes,
     serverless: {
       scaling: {
-        minpods: 0,
+        minpods: '',
         maxpods: '',
         concurrencytarget: '',
         concurrencylimit: '',
+        autoscale: {
+          autoscalewindow: '',
+          autoscalewindowUnit: '',
+          defaultAutoscalewindowUnit: 's',
+        },
+        concurrencyutilization: '',
```

Comment:
can we not use `serverlessInitialValues` here ?

Reply:
We have initial values defined for all the advanced options here. So added for these fields as well so that it matches the types